tests/OpenFOAM/OpenFOAM-8_motorBike_2M-cells_2cores.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+module load OpenFOAM/8-foss-2020a
+
+which ssh
+if [ $? -ne 0 ]; then
+    # if ssh is not available, set plm_rsh_agent to empty value to avoid OpenMPI failing over it
+    # that's OK, because this is a single-node run
+    export OMPI_MCA_plm_rsh_agent=''
+fi
+
+source $FOAM_BASH
+
+if [ -z $EBROOTOPENFOAM ]; then
+    echo "ERROR: OpenFOAM module not loaded?" >&2
+    exit 1
+fi
+
+export WORKDIR=/tmp/$USER/$$
+echo "WORKDIR: $WORKDIR"
+mkdir -p $WORKDIR
+cd $WORKDIR
+pwd
+
+# motorBike, 2M cells
+BLOCKMESH_DIMENSIONS="100 40 40"
+# motorBike, 150M cells
+#BLOCKMESH_DIMENSIONS="200 80 80"
+
+# X*Y*Z should be equal to number of available cores
+X=2
+Y=1
+Z=1
+# total number of cores
+NP=2
+# cores per node
+PPN=2
```

Comment:
`PPN=$NP`?

Reply:
In the long run, no, because `$NP` is the total number of cores, while `$PPN` is the number of cores per node.

Eventually we want to run this multi-node too.

Reply:
Ok, then I would define `NODES` first and use `$(( NP / $NODES ))` (after checking `$(( NP % NODES ))`)

Reply:
Or just `NP=$(($NODES * $PPN))`?

Reply:
Ah, no, because X*Y*Z determine *total* number of cores. Ugh.

Reply:
Ok, this would work without the need for rigorous testing (see https://stackoverflow.com/a/2395294) since it always rounds up:
```
PPN = $(( (NP + NODES -1)/NODES ))
```